plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -90,6 +90,12 @@ export class PersonState {
         return this.personContainer
     }
 
+    async updateExceptProperties(): Promise<LazyPersonContainer> {
+        await this.handleIdentifyOrAlias()
```

Comment:
Note that this merges properties mentioned in the RFC PR too, Ideally we'd only link the distinct_ids and merged the properties later, however that's the "fun" code we have, so maybe we can ignore this for now 🤷‍♀️

Reply:
Ok, I'll add a test for this case also.

Reply:
@tiina303 I'd expanded on the person properties test, it doesn't change the code however. Do you have a test scenario that would fail as is?

Reply:
Example scenario that would be different if we do properties merge before vs after:
```
identify(alice, {k: v1})
identify(bob, {k: v2})  -- or just identify(bob)
-- wait until processed (or just set up the persons that way before the test)
capture(bob, properties={$set: {k: v3}})  
create_alias(alice, bob)
-- these two would be in the buffer window, so 
1. if we process property merges early, we'll end up with the merge first which takes k=v1 from Alice (we merge into the first person), then process the property from identify, then we end up with overriding the value from the capture, i.e. getting k = v3
2. if we do property merges later, then we'll merge Alice with k=v1 and bob with k=v3 into Alice (keeping the value on conflict from the first person) we end up with k=v1
```

Reply:
Given that alice and bob are both "identified", I believe the alias will not work in this case. Should it work? I get this message:

```
🤔 refused to merge an already identified user via an $identify call
```

That's with the addition of this test: https://github.com/PostHog/posthog/pull/13505/commits/771d4b3f7d69780dfb01f9e0392d96627d0d8645

Reply:
aah well you can just do `custom events` for the first two too, the important thing was that they aren't in the buffer window

Reply:
Does this make sense now: https://github.com/PostHog/posthog/pull/13505/files#diff-e8c057ea332fa67a8f2bc6d811629e71d31b29630bf04fd906509be89f53e398R524